docs/src/use/migrate-to-10.0.0.md

```diff
@@ -18,6 +18,7 @@ The lists below are ordered roughly by the number of users each change is expect
 - [Node.js < v20.19, v21, v23 are no longer supported](#drop-old-node)
 - [New configuration file lookup algorithm](#config-lookup-from-file)
 - [`no-shadow-restricted-names` now reports `globalThis` by default](#no-shadow-restricted-names)
+- [`eslint:recommended` has been updated](#eslint-recommended)
```

Comment:
Not sure `eslint:recommended` is suitable - it may be confusing as it's like using eslintrc. what do think just something like "recommended rules"?